libs/libc/stdlib/lib_strtod.c

```diff
@@ -160,6 +160,7 @@ double strtod(FAR const char *str, FAR char **endptr)
     {
       set_errno(ERANGE);
       number = 0.0;
+      p = (FAR char *) str;
```

Comment:
```suggestion
      p = (FAR char *)str;
```

Reply:
Sorry, I just copied the existing style.

Its fixed now.